README.md

```diff
@@ -37,13 +44,61 @@ jobs:
 
 ## Inputs
 
-- `project-url` is the URL of the GitHub Project to add issues to.
-- `github-token` is a [personal access
+- <a name="project-url">`project-url`</a> _(required)_ is the URL of the GitHub Project to add issues to.
+- <a name="github-token">`github-token`</a> _(required)_ is a [personal access
   token](https://github.com/settings/tokens/new) with the `repo`, `write:org` and
   `read:org` scopes.
-- `labeled` is a comma-separated list of labels. For an issue to be added to the
-  project, it must have _one_ of the labels in the list. Omitting this key means
-  that all issues will be added.
+  _See [Creating a PAT and adding it to your repository](creating-a-pat-and-adding-it-to-your-repository) for more details_
+- <a name="labeled">`labeled`</a> _(optional)_ is a comma-separated list of labels used to filter applicable issues. When this key is provided, an issue must have _one_ of the labels in the list to be added to the project. Omitting this key means that any issue will be added.
```

Comment:
Took this improved input description from https://github.com/actions/add-to-project/pull/36, thanks @katblag ✨